src/Middleware/InjectDebugbar.php

```diff
@@ -84,11 +80,11 @@ public function handle($request, Closure $next)
      * (Copy from Illuminate\Routing\Pipeline by Taylor Otwell)
      *
      * @param $passable
-     * @param  Exception $e
+     * @param  Throwable $e
      * @return mixed
      * @throws Exception
      */
-    protected function handleException($passable, Exception $e)
+    protected function handleException($passable, $e)
```

Comment:
Couldn't `$e` be type-hinted as `Throwable`?

Reply:
Yeah you are right about that.

```suggestion
    protected function handleException($passable, Throwable $e)
```